### DISCUSSION TITLE
【求助】请问绳网怎么换头像

### DISCUSSION THREAD

**Discussion by minbvv** [share121/inter-knot#134] [其他]
试了一下在账号那边修改，但各种查询就是查不到哪里换头像，求大佬帮助

**Comment by unknown-sama-ff** [comment]
修改github的头像就行啦 账号信息里面

**Comment by AL-Pinecore** [comment]
在Github页面里改：
进入github.com，登陆你绳网使用的帐户后点击右上角头像，点击齿轮图标的Setting，右边有个Profile Picture和大大的头像，点击头像左下角的Edit，upload a picture就可以上传头像了

**Comment by minbvv** [comment]
感谢各位大佬的解答，目前已经成功换了头像，帖子就留着为后面有需求的小伙伴们留着了，另外再补充一下点击齿轮的设置之后要往下滑才能找到一个特别大的头像然后点击头像右下角的那个笔一样的按钮再点一下弹出来的那行字就可以修改了
